.github/workflows/ci.yml

```diff
@@ -31,3 +31,36 @@ jobs:
 
   vitest:
     uses: ./.github/workflows/vitest.yml
+
+  release:
+    needs: vitest
```

Comment:
I was thinking to make it depend on all the above jobs, but I should hope that you cannot merge in if they don't pass anyway, so depending on test only should be ok?

Reply:
Sure 👍 that's also consistent with how it's handled in other places, e.g. [in govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/blob/main/.github/workflows/ci.yml)